### PR TITLE
Update Service Providers page content (Level of Service + AAL options)

### PR DESCRIPTION
**Pull request by mmagsa**
### Relevant Ticket or Conversation:
JIRA issue [LG-6613](https://cm-jira.usa.gov/browse/LG-6613)
[login.gov Dashboard language updates - May 2022](https://docs.google.com/document/d/1R55WCfYScz4rm5g0SdIWo2mZQFyLTxGvkgX5CjNPwKs/edit)
### Description of Changes:
Modified Service Providers page sections "Level of Service" and "Default AAL Level" language as per above document:
#### Before
before1
before2
#### After
after1
after2
### PR Checklist:

1. [X] Have you linted and tested your code locally prior to submission?
2. [X] Have you tagged the appropriate dev(s) for review?
3. [X] Have you linked to any relevant tickets or conversations?

### PR Review Standards: 
- Consider using [Conventional Comments](https://conventionalcomments.org/) to ensure that your feedback is clear and actionable.
- Ideally, PRs should be reviewed by at least 2 team members.
- All PRs must be approved before being merged.

### CODE REVIEW

**Comment by NatashaJIbrahim** [comment]
@cdelivuk ya - frustratingly this content went through Content Review already :-/ @orenyk I know this was pushed through as being a bit urgent but I'm wondering if the workflow for content changes in the future should be through @cdelivuk so that we can make sure they don't add to our accessibility issues? 

I say we add it to our growing backlog in JIRA.

File: app/views/service_providers/_form.html.erb
Comment (orenyk) [inline app/views/service_providers/_form.html.erb:81]: **issue:** I don't think we should change the value to `1` here - with the way the IdP works, `nil` represents our default behavior. Funnily enough we do technically support a value of `1` but we never use it (since it wasn't available in the dropdown) - AAL1 is technically weaker than our default since it doesn't include MFA.

**Review by orenyk — CHANGES_REQUESTED** [review]
Looks good! A few comments about a) removing some unnecessary content and b) maybe cleaning up the duplicate strings using I18n. Let me know if you have any questions!

**Comment by orenyk** [comment]
@NatashaJIbrahim @cdelivuk I just saw your comments - totally agree that we should run these things through @cdelivuk moving forward! We can also definitely propose changes and get them through content approvals if we have usability concerns with the current length.

File: spec/features/service_providers_spec.rb
Comment (orenyk) [inline spec/features/service_providers_spec.rb:42]: **comment:** I think this assertion is passing because we say something like "we don't meet IAL2 at this time" in the help text for the level of service question, but it's not accurately checking that the correct option is selected. Do you want to update this assertion to also check for the correct `I18n` content and make sure that still passes?

**Review by orenyk — APPROVED** [review]
Looks great! One small comment that jumped on me but this is good to merge once that is resolved.

**Pull request merged**
